src/envoy/mixer/http_control.h

```diff
@@ -37,7 +37,8 @@ typedef std::shared_ptr<HttpRequestData> HttpRequestDataPtr;
 class HttpControl final : public Logger::Loggable<Logger::Id::http> {
  public:
   // The constructor.
```

Comment:
these two changes should not be here
1) you need to base your change to istio/proxy:rate_limting branch.  It seems that you are based on master branch.
2) or you need to sync rate_limiting branch to master